### ISSUE TITLE
False positive unresolved reference for `graphemes`

### ISSUE THREAD

**Issue by neonaot** [intellij-rust/intellij-rust#8686]
## Environment

* **IntelliJ Rust plugin version:** 0.4.168.4511-221-nightly
* **Rust toolchain version:** 1.61.0-nightly (f0c4da499 2022-03-01) x86_64-apple-darwin
* **IDE name and version:** CLion 2022.1 EAP (CL-221.5080.1)
* **Operating system:** macOS 11.4
* **Macro expansion engine:** new
* **Name resolution engine:** new
* **Additional experimental features:** org.rust.cargo.features.settings.gutter, org.rust.cargo.evaluate.build.scripts, org.rust.macros.proc

## Problem description
Code is working 
<img width="961" alt="image" src="https://user-images.githubusercontent.com/49211026/159726328-22a8b1a8-9428-4694-a5e7-0824bd768601.png">

## Steps to reproduce
```rust
#[cfg(feature = "grapheme")]
pub fn reverse_string(input: &str) -> String {
    input.graphemes(true).rev().collect()
}
```
cargo.toml:
```toml
[dependencies]
unicode-segmentation = "1.9.0"
[features]
grapheme = []
```
